src/browser/preload_scripts.ts

```diff
@@ -152,41 +139,14 @@ function fetchToCache(identity: Identity, preloadScript: PreloadScript): Promise
     });
 }
 
-// resolves to type `PreloadFetched` on success
-// resolves to `undefined` when fetch fails to download asset to Chromium cache
-function fetchPlugin(identity: Identity, preloadPlugin: PreloadPlugin): Promise<FetchResponse> {
-    return new Promise((resolve, reject) => {
-        const sourceUrl = getConfigUrlByUuid(identity.uuid);
-        const msg: PluginQuery = {
-            topic: 'application',
-            messageId: '1',
-            action: 'query-plugin',
-            name: preloadPlugin.name,
-            version: preloadPlugin.version,
-            optional: preloadPlugin.optional,
-            sourceUrl
-        };
-
-        rvmMessageBus.publish(msg, (resp) => {
-            if (resp.payload.hasOwnProperty('path') && resp.payload.action === 'query-plugin') {
-                const scriptPath = `${resp.payload.path}\\${resp.payload.target}`;
-                resolve({identity, preloadPlugin, scriptPath});
-            } else {
-                updatePreloadState(identity, preloadPlugin, 'load-failed');
-                resolve();
-            }
-        });
-    });
-}
-
 // resolves to type `PreloadLoaded` on success
 // resolves to `undefined` when above fetch failed or when successfully fetched asset fails to load from Chromium cache
 function loadFromCache(opts: FetchResponse): Promise<boolean> {
     return new Promise((resolve: Resolver, reject: Rejector) => {
         if (!opts || !opts.scriptPath) {
             resolve(false); // got fetchError above OR no error but no scriptPath either; in any case don't attempt to load
         } else {
-            const preload = opts.preloadScript ? opts.preloadScript : opts.preloadPlugin;
+            const preload = opts.preloadScript;
```

Comment:
`{ destructure }`?

Reply:
Tried to minimize changes to this file as this change was a quick one for other main changes in this PR.